### ISSUE TITLE
PaperWallet.CreateOrderOCO doesn't support buy side

### ISSUE THREAD

**Issue by ramilexe** [rodrigo-brito/ninjabot#256]
It seems `CreateOrderOCO` supports only sell option https://github.com/rodrigo-brito/ninjabot/blob/main/exchange/paperwallet.go#L409-L412

**Comment by rodrigo-brito** [comment]
The bot do not support OCO orders for futures market yet.
Futures Market are in an experimental phase.

**Comment by ramilexe** [comment]
@rodrigo-brito I am using PaperWallet with backtesting, not Binance Future

**Comment by rodrigo-brito** [comment]
Of course, we can try to work on that support, but we don't have that implemented right now. If you want to work on this feature, help is welcome
